### PR TITLE
test/rgw: add --gateways-per-zone to test_multi.py

### PR DESCRIPTION
**Pull request by cbodley**
for testing with some contention for the sync locks

### CODE REVIEW

**Comment by cbodley** [comment]
running this with --gateways-per-zone=2, i'm consistently hitting an assertion in RGWMetaSyncCR. see https://github.com/ceph/ceph/pull/10743

**Comment by cbodley** [comment]
this testing has uncovered another related issue: http://tracker.ceph.com/issues/17047

**Comment by oritwas** [comment]
lgtm

File: src/test/rgw/test-rgw-common.sh [hunk 1]
```diff
@@ -94,13 +94,11 @@ function init_first_zone {
   x $(rgw_admin $id) realm create --rgw-realm=$realm
 
 # create zonegroup, zone
-  x $(rgw_admin $id) zonegroup create --rgw-zonegroup=$zg --endpoints=http://localhost:$port --master --default
```
Comment (mikulely) [inline src/test/rgw/test-rgw-common.sh]: hi~ @cbodley It seems that we no longer set endpoints for zg,but instead just set endpoints on the master zone of zg? If so, maybe we need to update the doc https://github.com/ceph/ceph/blob/master/doc/radosgw/multisite.rst#creating-a-master-zonegroup

**Pull request merged**